Update metricbeat-event-structure.md (#47366)
elastic/beats: docs/reference/metricbeat/metricbeat-event-structure.md

:   Name of the module that the data is from

**`event.dataset`**

> :   Name of the dataset that the data is from

Reviewer: ```suggestion
:   Name of the source dataset in the format `<module>.<metricset>`
```
